.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,45 @@
+# Prerequisites
+
+```Please answer the following questions for yourself before submitting an issue. **YOU MAY DELETE THE PREREQUISITES SECTION.**```
+
+- [ ] I am running the latest version
+- [ ] I checked the documentation and found no answer
+- [ ] I checked to make sure that this issue has not already been filed
+- [ ] I'm reporting the issue to the correct repository (for multi-repository projects)
```

Comment:
Please change these statements to questions as you mentioned above.

Reply:
I'm sorry. I don't think I got that properly. Like, by questions do you mean something like : 

- [ ] Are you running the latest version ?
- [ ] Have you already checked out the documentation and were unable to find an answer ?
- [ ] and so on . . . 

Is this the expected format ?